sips/sip-026/sip-026-sbtc_peg.md

```diff
@@ -0,0 +1,171 @@
+# SIP-028: sBTC Bootstrap
+
+- **SIP Number:** 028
+- **Title:** A Decentralized and Programmable Asset Backed 1:1 with BTC 
+- **Consideration:** Technical, Economics
+- **Type:** Informational
+- **Status:** Draft
+- **Authors:**
+  - Andre Serrano
+  - Ashton Stephens
+  - Joey Yandle
+  - Mårten Blankfors
+  - Jesus Najera
+  - Jude Nelson
+  - Friedger Müffke
+  - Tycho Onnasch
+
+## Abstract
+sBTC is a SIP-010 token on the Stacks blockchain, backed 1:1 against BTC, and operated by a decentralized set of signers such that only a 70% majority can access the funds to maintain the protocol.
+
+While Bitcoin on its own cannot be used with smart contracts, sBTC provides a bridge of value from the Bitcoin Blockchain to the Stacks Blockchain to enable users who only own BTC to utilize the capability of smart contracts without switching to a new currency. This SIP aims to describe the high level sBTC system and the criteria for signer selection. 
```

Comment:
Typo: `Bitcoin Blockchain` --> `Bitcoin blockchain` (capitalization)

Reply:
"enable users who only own BTC to utilize the capability of smart contracts without switching to a new currency"

I think you need to make a choice.  Either this SIP proposes using sBTC as a gas token (in which case, a full-blown technical description of warranted),  or this SIP describes an auxiliary service that _could_ be built later with sBTC but is not part of this SIP.  If you want the latter, then this allusion should be dropped.

Reply:
Actually, ignore this.  Just use the abstract I have below.